packages/babel/cli.js

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env node
+const isGlobal = require("is-global");
+
+const globalMessage = isGlobal() ? " -g" : "";
 
 console.error("You have mistakenly installed the `babel` package, which is a no-op in Babel 6.\n" +
   "Babel's CLI commands have been moved from the `babel` package to the `babel-cli` package.\n" +
   "\n" +
-  "    npm uninstall babel\n" +
-  "    npm install babel-cli\n" +
+  "    npm uninstall" + globalMessage + " babel-cli\n" +
```

Comment:
For PRs with a few work-in-progress commits like this, we'll probably github's squash-and-merge behavior, so you are free to push another commit and it'll all get squashed in the end.

I don't have any other comments, looks solid :)